app/models/book.rb

```diff
@@ -1,15 +1,23 @@
 class Book < ActiveRecord::Base
   validates_presence_of :title, :published_date, :author, :price
 
-  delegate :sortable_fields, to: :class
+  scope :by_published, -> { order(published_date: :desc) }
 
+  self.per_page = 25
   def self.order_by(field, list_order)
-    list_order ||= "ASC"
-    field ||= "published_date"
-    Book.order("#{field} #{list_order}")
+    if field
```

Comment:
nil can be passed in here, which is why I have the if statement. Good call, though!